src/enums/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bcrs-shared-components/enums",
-  "version": "1.0.19",
+  "version": "1.0.20",
```

Comment:
Don't change this. The Lerna command in step 7 does this for you.

Please ensure you follow exactly the steps in the [README](https://github.com/bcgov/bcrs-shared-components) otherwise the versions can get messed, the branches can get messed up, or StoryBook can get messed up.